scripts/rollup/modules.js

```diff
@@ -230,6 +241,27 @@ function replaceDevOnlyStubbedModules(bundleType) {
   }
 }
 
+function replaceLegacyModuleAliases(bundleType) {
+  switch (bundleType) {
+    case UMD_DEV:
+    case UMD_PROD:
+      const modulesAlias = {};
+      for (var legacyModule in legacyModules) {
+        modulesAlias[legacyModule] = resolve(
```

Comment:
As per @gaearon's point above, if we put quotes around this, so it's `moduleAlias[\`'${legacyModule}'\`]` then the legacyModule order shouldn't matter as the replace module literally "just" does string replacement :)